modules/users/client/views/authentication/signup.client.view.html

```diff
@@ -26,6 +26,41 @@ <h3 class="col-xs-12 text-center">Sign up</h3>
           </div>
         </div>
         <div class="form-group" show-errors>
+          <label for="course">Course</label>
+          <select name="course" id="course" name="course" class="form-control" ng-model="vm.credentials.course" placeholder="TA" required>
+            <option value="course1">Course1</option>
+            <option value="course2">Course2</option>
+            <option value="course3">Course3</option>
+            <option value="course4">Course4</option>
+          </select>
+          <div ng-messages="vm.userForm.course.$error" role="alert">
+            <p class="help-block error-text" ng-message="required">Course is required.</p>
+          </div>
+        </div>
+        <div class="form-group" show-errors>
+          <label for="section">Section</label>
+          <select name="section" id="section" name="section" class="form-control" ng-model="vm.credentials.section" placeholder="TA" required>
+            <option value="section1">Section 1</option>
```

Comment:
Should not hard code these either. If we need to, we need to do this in the service that fetches these for us